include/wx/defs.h

```diff
@@ -2621,7 +2617,9 @@ typedef unsigned short  WXWORD;
 
 typedef WX_OPAQUE_TYPE(PicHandle ) * WXHMETAFILE ;
 
+#if !defined(__WXX11__) && !defined(__WXGTK__)
 typedef void*       WXDisplay;
+#endif
```

Comment:
Maybe we should just remove this one? I don't understand why would we need to have `WXDisplay` defined when building wxOSX, it should only be used when using X or GTK.

Reply:
I am fine with that. With wxGTK having it here creates a conflict, so removing it unconditionally works.

Reply:
Removed this one